_updates/2019-05-02-may-2019.md

```diff
@@ -0,0 +1,56 @@
+---
+title: May 2019
+---
+
+## Mononorepo changes
+- All packages build now with project references in packages folder (yarn build now runs tsc -b packages)
+  
+## Packages
+
+### [@sensenet/client-core@1.6.0](https://github.com/SenseNet/sn-client/releases/tag/%40sensenet%2Fclient-core%401.6.0)
+- Passing request inits to fetch
```

Comment:
There was a breaking change in client core. see https://github.com/SenseNet/sn-client/pull/152